spec/models/manageiq/providers/redhat/infra_manager/provision/state_machine_spec.rb

```diff
@@ -70,7 +70,6 @@ def test_poll_clone_complete
     @test_poll_clone_complete_setup ||= begin
       expect(task).to receive(:clone_complete?).and_return(false, false, true)
       expect(task).to receive(:requeue_phase).twice { requeue_phase }
-      expect(EmsRefresh).to receive(:queue_refresh).once
```

Comment:
@pkliczewski Was thinking maybe we should have converted this to a `expect(EmsRefresh).to_not receive(:queue_refresh)` to protect against that call being added back in.  What do you think?